### ISSUE TITLE
Allow tab completion when typing out branch names in the `checkout branch` and `delete branch` tabs.

### ISSUE THREAD

**Issue by H3RSKO** [GitGud-org/GitGud#103]
When a user goes to the Checkout or Delete branch tabs, they need to manually type out the full branch name.

You should be able to type just the first few letters and then hit tab to auto-complete the rest of the branch name.

**Comment by vnallamhawk** [comment]
is this still open ?

**Comment by H3RSKO** [comment]
Yup! Want to give it a go?

**Comment by sj056** [comment]
Hey @H3RSKO !
Is this issue still open and will it be considered for hacktoberfest?

**Comment by H3RSKO** [comment]
@sj056 Yes! This issue is open and an accepted PR merge would count for Hactoberfest.

**Comment by sj056** [comment]
Great! I'm working on this issue then.

**Comment by K-cloud25** [comment]
Hi @H3RSKO  
not sure if issue is active but created a Pull Request https://github.com/GitGud-org/GitGud/pull/106
Please let me know if it is a valid change. Only made changes to deleteBranch.js for test.
Thank you
